user/database-setup.md

```diff
@@ -182,34 +182,7 @@ before_script:
 
 ### PostgreSQL and Locales
 
-The following locales are installed on Travis CI build environements:
-
-- C
-- C.UTF-8
-- en_AG
-- en_AG.utf8
-- en_AU.utf8
-- en_BW.utf8
-- en_CA.utf8
-- en_DK.utf8
-- en_GB.utf8
-- en_HK.utf8
-- en_IE.utf8
-- en_IN
-- en_IN.utf8
-- en_NG
-- en_NG.utf8
-- en_NZ.utf8
-- en_PH.utf8
-- en_SG.utf8
-- en_US.utf8
-- en_ZA.utf8
-- en_ZM
-- en_ZM.utf8
-- en_ZW.utf8
-- POSIX
-
-You can find what language packs are currently available for Ubuntu 12.04 [on the packages site.](http://packages.ubuntu.com/search?keywords=language-pack&searchon=names&suite=precise§ion=all)
+The Travis CI build environment comes with a number of pre-installed locales, but you can also install additional ones, should you require them.
```

Comment:
Probably worth mentioning that install it using `apt` or whatever? wdyt

Reply:
The next paragraph covers the exact instructions for installing an extra-locale. I can add something like (_"see instructions below"_) maybe?

Reply:
:see_no_evil: